tests/unit-tests/gconstruct/test_transform.py

```diff
@@ -1147,7 +1148,122 @@ def test_hard_edge_dst_negative_transform(id_dtype):
     ground_truth[1][-2] = -1
     assert_equal(neg["hard_neg"][:,:10], ground_truth)
 
+@pytest.mark.parametrize("input_dtype", [np.cfloat, np.float32, np.float16])
```

Comment:
Do we have a test case for summation == 0?

Reply:
These is one in L1221